src/targets/gpu/compile_gen.cpp

```diff
@@ -180,12 +180,16 @@ std::string make_transformer_args(std::vector<std::string> transformers)
     return join_strings(std::move(transformers), ", ");
 }
 
-void generate_pointwise(cpp_generator& gg, const module& pm, const std::string& name)
+static void generate_pointwise(cpp_generator& gg,
+                               const module& pm,
+                               const std::string& name,
+                               bool always_return_tuple = false)
```

Comment:
You don need this if its already defaulted in the header

Reply:
This isn't defined in a header.

Reply:
[src/targets/gpu/include/migraphx/gpu/compile_gen.hpp](https://github.com/ROCm/AMDMIGraphX/pull/2957/files#diff-f2872ed1a864f2a138f87134aa2b21ed3f82f15db570dc7ac0807cb7e2528158)

Reply:
Actually my mistake on this one. But you're still flowing through false by default from the previous generate_pointwise from the header defaults, so is this false needed?

Reply:
No, its called in two places. It called in `generate_pointwise` but its also called in `generate_reduce` as well, and it doesnt pass the flag there.

Reply:
Fair. I see it. Now I see why you added the default.